src/DrawerLayout.js

```diff
@@ -82,10 +88,14 @@ export default class DrawerLayout extends Component {
   constructor(props: PropType, context: any) {
     super(props, context);
 
+    const windowWidth = Dimensions.get('window').width;
+
     this.state = {
       accessibilityViewIsModal: false,
       drawerShown: false,
       openValue: new Animated.Value(0),
+      windowWidth,
+      threshold: windowWidth / 2,
```

Comment:
Since threshold is always `this.state.windowWidth / 2`, it shouldn't be in the state, and should be calculated where it's used.